xamarin/Xamarin.Forms
# [iOS] Fix Keyboard Inset when Editor is one of the last components in a collection view

### Description of Change ###

I removed this code that disabled the native collectionview from adjusting correctly.
`CollectionView.ContentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentBehavior.Never;`

There was a comment that was above the line I removed that described the reason for having that line of code to begin with. I could not see any visible change when flipping my simulator into landscape mode w/ and w/o that line of code above - except for the fact that removing the code allowed me to see my editor's text. If someone could help explain, or include some repro steps in why it was there in the first place, that would be great as there may be side affects I'm unaware of from removing that code.

### Issues Resolved ### 
- fixes #9691

### API Changes ###
 
 None

### Platforms Affected ### 
- iOS

### Behavioral/Visual Changes ###

The user can now see what they've typed in an editor, other keyboard components.

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Go to the Issue9691 Test Case and ensure that when you click into the last Editor in the collection view, the view scrolls up and you can view the text you are entering into the editor.

### PR Checklist ###
